repository/meteorRepository.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+import { config } from "../config.js";
+
```

Comment:
you can get data from config by object destructor 
`const {nasaApiUrl, ...} = config` 
in this case you don’t have to write `config.key` each time